content/docs/user-guide/deploying/docker.md

```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
 pip install docker
 ```
 
-## TODO
+## Description
+
+Deploying to a docker container involves 2 main steps:
+
+1. Build the docker image.
+2. Start the server inside the image.
```

Comment:
Mention that you runs the image, to be more precise?

Reply:
Maybe "(by running `docker run` under the hood)"

Reply:
done